internal/namespaces/rdb/v1/custom_acl.go

```diff
@@ -92,9 +96,12 @@ func aclDeleteBuilder(c *core.Command) *core.Command {
 		}
 
 		aclDeleteResponse := aclDeleteResponseI.(*rdb.DeleteInstanceACLRulesResponse)
-		return rdb.ListInstanceACLRulesResponse{
-			Rules:      aclDeleteResponse.Rules,
-			TotalCount: uint32(len(aclDeleteResponse.Rules)),
+		for i := 0; i < len(aclDeleteResponse.Rules); i++ {
+			aclResult = append(aclResult, aclDeleteResponse.Rules[i].IP.String())
+		}
```

Comment:
`aclResult` can be initialized using `make([]string, 0, len(aclDeleteResponse.Rules)` as an optimization.